src/util.ts

```diff
@@ -784,6 +784,7 @@ export class Util {
     const options = {
       request: teenyRequest.defaults(requestDefaults),
       retries: autoRetryValue !== false ? maxRetryValue : 0,
+      noResponseRetries: autoRetryValue !== false ? maxRetryValue : 0,
```

Comment:
ah wow this is a separate config in the retry request library?

Reply:
Yes, `retry-request` has both https://github.com/stephenplusplus/retry-request/blob/f06f6351bd1f4d635804fb1c77e14984f2c9b2e4/index.js#L33 and https://github.com/stephenplusplus/retry-request/blob/f06f6351bd1f4d635804fb1c77e14984f2c9b2e4/index.js#L9